components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3508,6 +3508,163 @@ func TestListRules(t *testing.T) {
 	}
 }
 
+func TestListRulesForProject(t *testing.T) {
+	store, db, _ := setup(t)
+	defer db.close(t)
+	defer store.Close()
+
+	cases := map[string]func(*testing.T){
+		"when no rules or projects exist, returns an empty list": func(t *testing.T) {
+			ctx := context.Background()
+			resp, err := store.ListRulesForProject(ctx, "not-found")
+			assert.NoError(t, err)
+			assert.Nil(t, resp)
+			assert.Equal(t, 0, len(resp))
```

Comment:
[nit] This could also be `assert.Zero(t, len(resp))`. Doesn't matter. 😄

Reply:
If it matters to you, it matters!

Reply:
Fixed in a few other spots as well.